src/basic.rs

```diff
@@ -4,26 +4,46 @@ use std::{cmp::Ordering, unreachable};
 // NOTE: `neg`, `add`, `sub`, `mul` and `div` are implemented in arith.rs
 
 impl Interval {
-    /// When `self` and `rhs` are bounded, returns the tightest
-    /// interval `z` such that `rhs` + `z` ⊇ `self` if such `z`
-    /// exists, which is the case iff the width of `self` is greater
-    /// than or equal to the width of `rhs`.  If `self` or `rhs` is
-    /// unbounded, or such `z` does not exist, returns
-    /// [`ENTIRE`][Self::ENTIRE].
+    /// Returns the *unique* interval `z` such that `rhs + z` ⊇ `self`
```

Comment:
There is *not* a *unique* interval that satisfies `rhs + z` ⊇ `self`.  That's why the norm says the *tightest*.

Reply:
Wow… it's so dumb mistake. Thank you so much for pointing it out! I revert the change now, and I'll ask for your review when I make another change to the doc.